Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -472,12 +589,45 @@ public override void Update(float elapsedClockSeconds)
             // triple valve is set to charge the brake cylinder
             if ((TripleValveState == ValveState.Apply || TripleValveState == ValveState.Emergency) && !Car.WheelBrakeSlideProtectionActive)
             {
-                float dp = elapsedClockSeconds * MaxApplicationRatePSIpS;
+                float dp = 0;
+                float dpPipe = 0;
+                float BrakePipeChange = Math.Max(prevBrakePipePressurePSI - BrakeLine1PressurePSI, 0);
```

Comment:
```suggestion
                float brakePipeChange = Math.Max(prevBrakePipePressurePSI - BrakeLine1PressurePSI, 0);
```
Please use lowercase names for local variables